org.OpenT2T.Sample.SuperPopular.Thermostat/org.OpenT2T.Sample.SuperPopular.Thermostat.xml

```diff
@@ -10,10 +10,20 @@
     <method name="turnOff"/>
     
     <!-- Get the current ambient temperature -->
-    <method name="getCurrentTemperature">
+    <method name="getAmbientTemperature">
+      <!-- Temperature scale:  'f' for Fahrenheit. 'c' for celsius -->
+      <arg name="scale" type="s" direction="in" />
       <arg name="temp" type="d" direction="out" />
     </method>
 
+    <!-- Set the current ambient temperature -->
+    <method name="setTargetTemperature">
```

Comment:
I don't see a translator implementation for the get/set Heating/Cooling setpoint methods, so I thought you were adding setTargetTemperature instead of those. But apparently it is a 3rd (middle?) target temperature that Nest defines in addition to the usual high/low targets? How does that work?

---
In reply to: [68282643](https://github.com/openT2T/translators/pull/22#discussion_r68282643) [](ancestors = 68282643)